pkg/model/components/containerd.go

```diff
@@ -41,64 +42,58 @@ func (b *ContainerdOptionsBuilder) BuildOptions(o interface{}) error {
 
 	containerd := clusterSpec.Containerd
 
-	if fi.StringValue(clusterSpec.Containerd.Version) == "" {
-		containerdVersion := ""
-
-		if clusterSpec.ContainerRuntime == "containerd" {
-			if clusterSpec.KubernetesVersion == "" {
-				return fmt.Errorf("Kubernetes version is required")
-			}
+	if clusterSpec.ContainerRuntime == "containerd" {
+		if b.IsKubernetesLT("1.11") {
+			// Containerd 1.2 is validated against Kubernetes v1.11+
+			// https://github.com/containerd/containerd/blob/master/releases/v1.2.0.toml#L34
+			return fmt.Errorf("kubernetes %s is not compatible with containerd", clusterSpec.KubernetesVersion)
+		} else if b.IsKubernetesLT("1.18") {
+			klog.Warningf("kubernetes %s is untested with containerd", clusterSpec.KubernetesVersion)
```

Comment:
This might be a little strong, in that it's the combination kops + kubernetes that we haven't tested before 1.18, but we can let it stand :-)